gap/lqueue.gi

```diff
@@ -21,7 +19,7 @@ function(arg)
 
   result := [1, 1, capacity, EmptyPlist(capacity)];
   t := NewType(CollectionsFamily(FamilyObj(IsObject)),
-               filter and IsPositionalObjectRep);
+               filter and IsMutable and IsPositionalObjectRep);
```

Comment:
The commit with message  "Test PlistQueue implementation" does lots of things not covered by its commit message.